src/uu/tail/Cargo.toml

```diff
@@ -29,6 +29,9 @@ fundu = { workspace=true }
 windows-sys = { workspace=true, features = ["Win32_System_Threading", "Win32_Foundation"] }
 winapi-util = { workspace=true }
 
+[dev-dependencies]
+rstest = "0.17.0"
```

Comment:
Shouldn't this be a workspace dependency?

Reply:
yeah, that's better. I overread in the docs that [dev-dependencies]` can be workspace dependencies, too.